packages/botonic-plugin-flow-builder/src/action.tsx

```diff
@@ -57,3 +57,16 @@ export class FlowBuilderAction extends React.Component<FlowBuilderActionProps> {
     return contents!.map(content => content.toBotonic(content.id))
   }
 }
+
+export class FlowBuilderMultichannelAction extends FlowBuilderAction {
+  render(): JSX.Element | JSX.Element[] {
+    // @ts-ignore
+    const { contents, handoffNode } = this.props
+    return (
+      //@ts-ignore
+      <Multichannel text={{ buttonsAsText: false }}>
```

Comment:
we should think how we configure the multichannel from the outside in the case of flow-builder-plugin. for now it's okay